osu!stream/Resources/Resource.designer.cs

```diff
@@ -0,0 +1,8698 @@
+#pragma warning disable 1591
+//------------------------------------------------------------------------------
+// <auto-generated>
+//     This code was generated by a tool.
+//
+//     Changes to this file may cause incorrect behavior and will be lost if
+//     the code is regenerated.
+// </auto-generated>
+//------------------------------------------------------------------------------
+
+[assembly: global::Android.Runtime.ResourceDesignerAttribute("osum.Resource", IsApplication=true)]
```

Comment:
Is this file actually required? If it can be removed, it would aid in me reviewing this PR more easily.

Reply:
> If it is required, at least try and reduce the number of lines. Right now it looks to contain a lot of resources which don't need to be used by the game.

The file is regenerated by every build, so i added it to the .gitignore